README.adoc

```diff
@@ -75,47 +77,76 @@ is in the `start/src` directory:
             └── WEB-INF
 ----
 
-== Installing Gradle
+== Testing Gradle
 
-If you don't have Gradle Version 3 or higher installed, follow the instructions on the 
-https://gradle.org/install/[Gradle installation page^] for your operating system.
-Configure the environment variables so that you can run the `gradle` command.
+If you do not have Gradle installed, running the Gradle Wrapper will automatically install Gradle. The Gradle 
+Wrapper files are included in the project directory. Learn more about Gradle Wrapper on the 
+https://docs.gradle.org/current/userguide/gradle_wrapper.html[official website^].
 
 Test that Gradle is installed properly by running the following command in a command line:
```

Comment:
Change to
"Test that Gradle is installed properly by running the following command:"

Reply:
Addressed.